src/tiffcomposite_int.hpp

```diff
@@ -1030,13 +1024,10 @@ namespace Exiv2 {
         uint32_t doSizeImage() const override;
         //@}
 
-    private:
-        //! @name NOT implemented
-        //@{
-        //! Assignment operator.
-        TiffSubIfd& operator=(const TiffSubIfd& rhs);
-        //@}
+    public:
+        TiffSubIfd& operator=(const TiffSubIfd& rhs) = delete;
```

Comment:
Are the move assignment operators missing here intentionally?

Reply:
Addressed by 69e6b7e117243ac95288c76f6c608bfb0f640554.